content/inference/particlenet.md

```diff
@@ -86,35 +86,48 @@ Recent works in the joint field of HEP and ML also shed light on exploiting the
 
 Now we walk through three solid examples to get you familiar with `Weaver`. We use the benchmark of the top tagging task [[arXiv:1707.08966](https://arxiv.org/abs/1707.08966)] in the following example. Some useful information can be found in the "top tagging" section in the [IML public datasets webpage](https://iml.web.cern.ch/public-datasets) (the [gDoc](https://docs.google.com/document/d/1Hcuc6LBxZNX16zjEGeq16DAzspkDC4nDTyjMp1bWHRo/edit)).
 
-Our goal is to do some warm-up with `Weaver`, and more importantly, to explore from a technical side the neural net architectures, from a simple multi-layer perceptron (MLP) model, to a more complicated "DeepAK8 tagger" model based on 1D CNN with ResNet, and eventually to the "ParticleNet model" which is based on DGCNN. We will dig deeper into their implementations in `Weaver` and try to illustrate as many details as possible. Finally, we compare their performance and see if we can reproduce the benchmark record with the model. Please clone the repo [`weaver-benchmark`](https://github.com/colizz/weaver-benchmark) and we'll get started.
+Our goal is to do some warm-up with `Weaver`, and more importantly, to explore from a technical side the neural net architectures, from a simple multi-layer perceptron (MLP) model, to a more complicated "DeepAK8 tagger" model based on 1D CNN with ResNet, and eventually to the "ParticleNet model" which is based on DGCNN. We will dig deeper into their implementations in `Weaver` and try to illustrate as many details as possible. Finally, we compare their performance and see if we can reproduce the benchmark record with the model. Please clone the repo `weaver-benchmark` and we'll get started. The `Weaver` repo will be cloned as a submodule.
```

Comment:
Our goal is to do some warm-up with `Weaver`, and more importantly, to explore from a technical side the neural net architectures: a simple multi-layer perceptron (MLP) model, a more complicated "DeepAK8 tagger" model based on 1D CNN with ResNet, and the "ParticleNet model," which is based on DGCNN.